### DISCUSSION TITLE
improving ESpeak-ng for NVDA

### DISCUSSION THREAD

**Discussion by joshknnd1982** [nvaccess/nvda#13822] [Ideas]
In future versions of NVDA, so people can tollerate ESpeak-ng better, Please do the following... 1. set ESpeak to Edward Klatt voice by default. 2. Set pitch to 70, inflection to 30. 3. Lower the headsize slightly, make the voice sound a bit bigger and let the user adjust voice headsize. 3. the way klatt voices  pronounces the N sound in words is too closed like in the French language. its more like a french n so please adjust the Klatt voices so n has a more open sound.

**Comment by trypsynth** [comment]
I think the current defaults are honestly fine, especially seeing as NVDA's default isn't eSpeak anymore, its OneCore.

All the settings you listed are very subjective. I find the defaults to be quite reasonable.

Also, the way eSpeak says things is not up to NVDA, that's up to eSpeak. Consider opening an issue there.

**Comment by Brian1Gaff** [comment]
I like Espeak over the natural voices for the menus and that sort of thing, 
but the real voice for reading documents. This unfortunately means 
attempting to make pronunciations agree by tweaking dictionaries.
 I would suggest that the current default voice used by Espeak say when 
running on Windows 7 and being run elsewhere is not the best one, so would 
support a different voice, One that is less shrill but still has good top 
end. I like Quincy, but there are others, and often its what people hear 
first that gives them the impression of the synth. Maybe making it sound 
more neutral would do this better than the default one.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by trypsynth** [comment]
I think a decent default is how Orca does it. Voice none, rate 50, pitch 50, inflection 50.

**Comment by Jiminator** [comment]
How do I change inflection on espeak-ng
